charts/lakefs/values.yaml

```diff
@@ -92,3 +92,14 @@ extraEnvVars:
 #   successThreshold: 4
 #   timeoutSeconds: 1
 #   initialDelaySeconds: 5
+
+# Everything under fluffy stanza is for lakeFS enterprise only https://docs.lakefs.io/enterprise/
+fluffy:
+  enabled: false
+  image:
+    repository: treeverse/fluffy
+    tag: '0.2.0'
+    pullPolicy: IfNotPresent
+
+# Start local postgres pod for quick start, not for production
+useDevPostgres: true
```

Comment:
Not like we discussed -
I'm on that `useDevPostgres` will be explicit `true` - I just think we should condition the setup and deploy of it in case rbac is `false`.